pkgs/by-name/ma/mago/package.nix

```
@@ -0,0 +1,38 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  testers,
+  mago,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "mago";
+  version = "0.0.5";
+
+  src = fetchFromGitHub {
+    owner = "carthage-software";
+    repo = pname;
```

```suggestion
    repo = "mago";
```
https://github.com/NixOS/nixpkgs/issues/277994

---

done